src/Swarm/TUI/View.hs

```diff
@@ -1236,6 +1240,66 @@ displayProperties = displayList . mapMaybe showProperty
       , txt " "
       ]
 
+-- | This widget can have potentially multiple "headings"
+-- (one per capability), each with multiple commands underneath.
+-- Directly below each heading there will be a "exercise cost"
+-- description, unless the capability is free-to-exercise.
+explainCapabilities :: GameState -> Entity -> Widget Name
+explainCapabilities gs e
+  | null capabilitiesAndCommands = emptyWidget
+  | otherwise =
+      padBottom (Pad 1) $
+        vBox
+          [ hBorderWithLabel (txt "Enabled commands")
+          , hCenter
+              . vBox
+              . L.intersperse (padTop (Pad 1) . hCenter . txt $ T.replicate 10 "*")
```

Comment:
I'm not a big fan of the asterisk dividers; they don't seem to fit the rest of the visual aesthetic and they waste vertical space.  Do we need dividers between the command descriptions at all?  I think just a blank line of space between commands ought to be fine.

Reply:
Fixed.  Sections are now delimited by two blank lines.   I think indentations do a sufficient job delimiting the other content.

![Screenshot from 2024-04-24 22-54-18](https://github.com/swarm-game/swarm/assets/261693/561ee8cb-9359-479a-94da-5c8266a0874b)